src/webots/core/WbProject.cpp

```diff
@@ -36,6 +37,7 @@ static QString gPreviousPath = QString();
 static WbProject *gCurrentProject = NULL;
 static WbProject *gSystemProject = NULL;
 static WbProject *gDefaultProject = NULL;
+static WbProject *gextraDefaultProject = NULL;
```

Comment:
```suggestion
static WbProject *gExtraDefaultProject = NULL;
```

Reply:
:+1: (looks like I forget to check 'match case' in my search and replace...)